nosekatreport/__init__.py

```diff
@@ -2,9 +2,17 @@
 # BSD license - see LICENSE for details
 
 """Root of the nosekatreport package."""
+from __future__ import (absolute_import, division, print_function)
+
+from future import standard_library
+standard_library.install_aliases()
 
 # BEGIN VERSION CHECK
 # Get package version when locally imported from repo or via -e develop install
+
```

Comment:
Why did you move the imports?

See regarding imports [CAM Developer's Guide](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit#heading=h.y0sdkqkmrbb3)

Reply:
This resulted from the futurize stage 0 application.

Reply:
Kindly fix this, please. Follow the guideline else refer below

We expand this to:
- standard library imports
- related third party imports
- ‘kat’ imports, i.e. packages that are in SVN
- **local application/library specific imports**	

> "The first rule of any technology used in a business is that automation applied to an efficient operation will magnify the efficiency. The second is that automation applied to an inefficient operation will magnify the inefficiency." - Bill Gates

**Note:** Automate and make manual fixes were needed.

Reply:
Latest Commit addresses this change.